### PR TITLE
dropdown menu for adding information pages and minutes

### PR DESCRIPTION
**Pull request by janno42**
this adds a dropdown menu entry for creating new information pages and minutes.

not part of this pr: recovering previously created minutes documents when creating new ones (similar to what is done for information pages).

### CODE REVIEW

**Comment by invliD** [comment]
Please rebase. #211 fixed the build issue.

File: _1327/minutes/forms.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+from guardian.shortcuts import get_perms_for_model, assign_perm, remove_perm
+
+from _1327.user_management.models import UserProfile
+from _1327.documents.forms import DocumentForm, StrippedCharField
+from .models import MinutesDocument
+
+
+class MinutesDocumentForm(forms.ModelForm):
```
Comment (Nef10) [inline _1327/minutes/forms.py]: remove current MinutesDocumentForm from minutes/models.py

Comment (janno42) [inline _1327/minutes/forms.py]: done

**Comment by Nef10** [comment]
:shipit:

**Pull request merged**